whylog/config/__init__.py

```diff
@@ -127,11 +158,12 @@ def _convert_parsers_to_file_form(self, parser_definitions):
 
 
 class InvestigationPlan(object):
-    def __init__(self, front_input, rule_subset, log_location_dict):
-        pass
+    def __init__(self, suspected_rules, investigation_metadata):
+        self._suspected_rules = suspected_rules
+        self._investigation_metadata = investigation_metadata
 
-    def get_next_investigation_step(self):
-        pass
+    def get_next_investigation_step_with_log_type(self):
+        return self._investigation_metadata[0]
```

Comment:
this should remove the step from the front of the queue or it will keep returning the first element over and over